Dolphin/windWakerInterface.py

```diff
@@ -1,57 +1,57 @@
 import random
-
 import dolphin_memory_engine as dme
-
+from typing import List
 import Dolphin.windWakerResources as WWR
 from Model.config import Config
 
 random_rupoors = Config.get_config().Random_Rupoors
 
-def hook():
+
+def hook() -> None:
     dme.hook()
 
 
-def assert_hook():
+def assert_hook() -> None:
     dme.assert_hooked()
 
 
-def is_hooked():
+def is_hooked() -> bool:
     return dme.is_hooked()
 
 
 def write_byte(address, value: int) -> None:
     dme.write_byte(address, value)
 
 
-def read_byte(address):
+def read_byte(address) -> None:
```

Comment:
This returns an `int`.

For all the `read`/`write` functions, `address` is an `int`, and `value` is going to be an `int`.
I'll be noting the exceptions to this policy with comments.